doc/api/assert.md

```diff
@@ -621,8 +625,9 @@ are also recursively evaluated by the following rules.
   both sides are `NaN`.
 * [Type tags][Object.prototype.toString()] of objects should be the same.
 * Only [enumerable "own" properties][] are considered.
-* [`Error`][] names and messages are always compared, even if these are not
-  enumerable properties.
+* [`Error`][] names, messages and causes are always compared, even if these are
+  not enumerable properties. For `AggregateError`, non-enumerable property
```

Comment:
I believe the actual behavior unconditionally compares the `errors` property regardless of whether the error instance is an `AggregateError` or not. This could be reworded as

```
* [`Error`][] names, messages, causes, and errors are always compared, even if these are not enumerable properties.
```

Reply:
Resolved. Thanks for the reminder